controllers/data.controller.js

```diff
@@ -89,7 +89,17 @@ const handlePlacementQuestions = async (
       },
     },
   );
-  return response;
+  if (response) {
+    response = {
+      status: { code: 200, message: "Score and tier updated successfully" },
```

Comment:
i dont think you need to pass status here. it is just helper function, you can just get the data and check it at updateTierAndScore function.